submit_to_archives.py

```diff
@@ -71,18 +82,37 @@ def get_header():
 
 
 def process_archiver_response():
-    response_text = archiver_response.json()
-    submitted_samples_in_biosamples =\
-        [f'{BIOSAMPLES_SAMPLE_URL}/{sample_accession}' for sample_accession in response_text.get('biosamples_accessions')]
-    submitted_project_in_biostudies =\
-        [f'{BIOSTUDIES_STUDY_URL}{response_text.get("biostudies_accession")}']
-    submitted_entities_in_ena =\
-        [f'{ENA_WEBIN_BASE_URL}{ena_accession}' for ena_accession in response_text.get('ena_accessions')]
-    return {
-        'Submitted samples in BioSamples': submitted_samples_in_biosamples,
-        'Submitted studies in BioStudies': submitted_project_in_biostudies,
-        'Submitted entities in ENA': submitted_entities_in_ena
-    }
+    response_text = response_json
+
+    output = {}
+    for archive_name, archived_items in response_text.items():
```

Comment:
There a re many nesting levels and conditionals here. This would make it difficult to understand, test and debug.

Reply:
Refactored